shared-helpers/src/formKeys.ts

```diff
@@ -178,3 +178,205 @@ export const preferredUnit = [
 ]
 
 export const bedroomKeys = ["studio", "oneBdrm", "twoBdrm", "threeBdrm"]
+
+export const neighborhoodKeys = [
```

Comment:
Might be good to alphabetize this list.

Reply:
this list gets alphabetized with the sort function in the next line so in the dropdown the list is alphabetized, but should I still alphabetize it in the array as well for readability?